assets/js/custom/TranslateProgram.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-env jquery */
+/* global Translation */
+
+// eslint-disable-next-line no-unused-vars
+class TranslateProgram extends Translation {
+  constructor (translatedByLine, programId, hasDescription, hasCredit) {
+    super(translatedByLine)
+    this.programId = programId
+    this.hasDescription = hasDescription
+    this.hasCredit = hasCredit
+    this._initListeners()
+  }
+
+  _initListeners () {
+    const self = this
+    $(document).on('click', '#program-translation-button', function () {
+      $(this).hide()
+
+      if (self.isTranslationNotAvailable('#name-translation')) {
+        $('#program-translation-loading-spinner').show()
+        self.translateProgram()
+      } else {
+        self.openTranslatedProgram()
+      }
+    })
+
+    $(document).on('click', '#remove-program-translation-button', function () {
+      $(this).hide()
+      $('#program-translation-button').show()
+
+      $('#name').removeClass('program-name').addClass('program-name-animation')
+      $('#name-translation').removeClass('program-name').addClass('program-name-animation')
+      $('#name-translation').animate({ width: 'toggle' })
+      $('#name').animate({ width: 'toggle' }, 400,
```

Comment:
400 is the time of the animation, I needed to add it since the function() parameter has to be the third parameter in the function call. 400 is the standard time, so the animation would run in exactly the same time if this parameter wasn't specified.

Reply:
Can you add 400 as a constant to improve readability?

Reply:
Yup, I added const animationTime to the function

Reply:
not sure what the naming conventions are for constants in js files, maybe should be `ANIMATION_TIME`? also you defined this twice, you can define it just once on top of the file.

Reply:
Updated to ANIMATION_TIME